app/public/css/768-1024px.css

```diff
@@ -32,6 +32,10 @@
     margin: 16px 0px -42px;
   }
 
+  .helper-text-wrapper {
+    margin-bottom: -37px;
```

Comment:
Nitpick: I think the policy is that the px should be multiple of 4 or at least an even number. If it looks worse with an even number then feel free to ignore this. Although, I think the "Taking too long" text could be a pixel or two close to the helper text and farther from the bottom.